src/style.css

```diff
@@ -1,3 +1,63 @@
+:root {
+  --blue-color: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  text-transform: uppercase;
+}
+
+.menu {
+  margin-top: 0;
+  margin-right: 0;
+  background-color: white;
+  padding: 0;
+  border: 0;
+  display: flex;
+  justify-content: flex-end;
+  justify-content: space-between;
+}
+
+.menu:hover {
+  border: 1px solid var(--blue-color);
+}
+
+.logo {
+  margin-top: 10px;
+  margin-left: 50px;
+  float: left;
+}
+
+.nav {
+  display: flex;
+  margin-top: 0;
+  margin-right: 0;
+  height: 60px;
+  justify-content: flex-end;
+}
```

Comment:
no need to style nav heve, it's just a container, it would make more sense to style your `<ul>` element, so try to add styles for `nav__list`